src/applications/personalization/dashboard/actions/messaging.js

```diff
@@ -15,49 +15,61 @@ import {
 
 const baseUrl = `${environment.API_URL}/v0/messaging/health`;
 
-export function fetchFolder(id, query = {}) {
-  return dispatch => {
-    const errorHandler = () => dispatch({ type: FETCH_FOLDER_FAILURE });
+export const fetchFolder = (id, query = {}) => async dispatch => {
```

Comment:
Most of the changes to this file are just a cleanup to make the code more readable. This was it should be easier to follow the success/error logic.